addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -49,8 +61,8 @@ export class MassMailingHtmlField extends HtmlField {
             onIframeUpdated: () => this.onIframeUpdated(),
             snippets: 'mass_mailing.email_designer_snippets',
             resizable: false,
-            defaultDataForLinkTools: { isNewWindow: true },
-            toolbarTemplate: 'mass_mailing.web_editor_toolbar',
+            linkInitialIsNewWindow: true,
```

Comment:
Did you also do it for the colorpalette ? https://github.com/odoo/odoo/pull/118966#discussion_r1229140228

Reply:
No because the options are not given to a ColorPalette, it is used directly in the component.
Do you want me to still change it?

Like so:
```js
        onColorpalette: {
            type: Object,
            shape: {
              dropdownShow: { type: Function, optional: true },
              dropdownHide: { type: Function, optional: true },
            },
            optional: true,
        }
        textColorPaletteProps: { type: Object },
        backgroundColorPaletteProps: { type: Object },
```